management_app/templates/faculty/edit.html

```diff
@@ -0,0 +1,131 @@
+{% set is_active = faculty['profile_status']['active_status'] %}
+<div>
+  <h4>Year: {{ year }}-{{ year+1 }}</h4>
+  <ul>
+    <li>Previous Year Teaching Point Balance: <b>{{ faculty['previous_balance'] }}</b></li>
+    <li>Current Ending Year Teaching Point Balance: <b>{{ faculty['ending_balance'] }}</b></li>
+    <!-- TODO: get new ending from front-end before send POST request-->
+    <!-- <li>New Ending Year Teaching Point Balance: <b id="new_ending_balance"></b></li> -->
+  </ul>
+</div>
+<form method="POST" enctype="multipart/form-data" class="col-lg-6 col-sm-12" style="padding-bottom: 60px;">
+  <div class="mb-3">
+    <label class="fw-bold" for="name">Name*</label>
+    <input name="name" type="text" class="form-control" id="name" placeholder="Anteater Smit" value="{{ request.form['name'] or faculty['name'] }}" required>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="ucinetid">UCI NetID*</label>
+    <input name="ucinetid" type="text" class="form-control" id="ucinetid" placeholder="anteater1" value="{{ request.form['ucinetid'] or faculty['ucinetid'] }}" required>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="email">Email address*</label>
+    <input name="email" type="email" class="form-control" id="email" placeholder="anteater1@uci.edu" value="{{ request.form['email'] or faculty['email']  }}" required>
+  </div>
+  <div class="mb-3 ">
+    {% set faculty_role = faculty['profile_status']['role'] %}
+    <div class="d-flex">
+      <label class="fw-bold" class="align-self-center" for="role" style="width: 160px;">Faculty Role*&nbsp;</label>
+      <select class="form-select align-self-center" id="role" aria-label="Faculty Role" name="role" required onchange="handleRoleSelection(this)">
+        {% for option, point in role_options.items() %}
+        <option
+          value="{{ option }}" data-point="{{ point }}" id="user_role_option"
+          {% if faculty_role== option %} selected="selected"{% endif %}
+        >
+          {{ option }}
+        </option>
+        {% endfor %}
+      </select>
+    </div>
+    <p>Please update faculty role on the edit page for the latest academic year for faculty member you would like to update.</p>
+    <ul>
+      <li>Required Credit Due: <span id="user_role_credit_due">{{ faculty['credit_due'] }}</span></li>
+    </ul>
+  </div>
+  <div class="mb-3">
+    Faculty Role History in Current System:
+    <ul>
+    {% for f in roles_status %}
+      {% if f['active_status'] %}
+      <li>
+        {{ f['start_year'] }} - {{ f['end_year'] }}: {{ f['role'] }}
+      </li>
+      {% endif %}
+    {% endfor %}
+    </ul>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="grad_count">Grad Count*</label>
+    <input name="grad_count" type="text" class="form-control" id="grad_count" placeholder="3.33" value="{{ request.form['grad_count'] or faculty['grad_count']  }}" required>
+    <p>If no grad count, please type 0.</p>
```

Comment:
to match the placeholder, maybe change it to "please type 0.0"

Reply:
Updated to please type 0.0.